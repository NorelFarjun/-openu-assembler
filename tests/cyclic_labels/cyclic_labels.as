L1: lea L2, r1
L2: lea L1, r2
