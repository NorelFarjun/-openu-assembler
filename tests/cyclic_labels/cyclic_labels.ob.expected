   6 0
0100 20504
0101 01472
0102 00014
0103 20504
0104 01442
0105 00024
